### PR TITLE
add support in Bundle easyblock to run full sanity check

### PR DESCRIPTION
**Pull request by boegel**
This is required for bundles like `R-bundle`, to make sure the sanity check for the extensions is run.

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by wpoely86** [comment]
looks fine but can you not make it run the sanity check automatically when extensions are used?

**Comment by boegel** [comment]
@wpoely86: Hmm, good idea. Also when `sanity_check_paths` or `sanity_check_commands` are given...

**Comment by boegel** [comment]
@wpoely86: please rereview?

File: easybuild/easyblocks/x/xmipp.py [hunk 1]
```diff
@@ -67,14 +67,19 @@ def configure_step(self):
...
 
-        # extract some dependencies that we really need and can't find anywhere else
+        # extract some dependencies that we really need and can't find anywhere else, if needed
         # alglib tarball has version in name, so lets find it with a glob
         # we can't do this in extract step before these are in the original sources tarball, so we need to know
         # startdir first
         external_path = os.path.join(self.cfg['start_dir'], 'external')
-        alglib_tar = glob.glob(os.path.join(external_path, 'alglib*.tgz'))[0]
-        for src in ['bilib.tgz', 'bilib.tgz', 'condor.tgz', alglib_tar, 'scons.tgz']:
-            extract_file(os.path.join(external_path, src), external_path)
```
Comment (wpoely86) [inline easybuild/easyblocks/x/xmipp.py]: is this related in some way?

**Comment by boegel** [comment]
@wpoely86: please rereview?

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyblocks/x/xmipp.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/x/xmipp.py]: argh, crap, no, it's not :(

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!
